packages/block-library/src/missing/edit.native.js

```diff
@@ -34,6 +34,7 @@ import styles from './style.scss';
 
 // Blocks that can't be edited through the Unsupported block editor identified by their name.
 const UBE_INCOMPATIBLE_BLOCKS = [ 'core/block' ];
+const I18N_BLOCK_SCHEMA_TITLE = 'block title';
```

Comment:
This value was taken from [`packages/blocks/src/api/i18n-block.json` file](https://github.com/WordPress/gutenberg/blob/feecdae6210a14d166001728dd284252bc01ffed/packages/blocks/src/api/i18n-block.json#L2).